.github/actions/publish/action.yml

```diff
@@ -45,11 +45,19 @@ runs:
         # specify the same version as in ~/.python-version
         python_version: "3.9.11"
         pip_version: "21.1"
-        subdir: "protocol-models/python/"
+        subdir: "protocol-models/python/airbyte_protocol"
       env:
         TWINE_PASSWORD: ${{ inputs.twine_password }}
         TWINE_USERNAME: ${{ inputs.twine_username }}
 
+    - name: Publish Pydantic-v2 Python Package
+      uses: mariamrf/py-package-publish-action@v1.1.0
+      with:
+          # specify the same version as in ~/.python-version
```

Comment:
Probably not useful for others?

Reply:
eh was just a copy paste, and good to know where it came from